docs/user_guide.md

```diff
@@ -0,0 +1,66 @@
+   :target: https://mybinder.org/v2/gh/alan-turing-institute/scivision/HEAD?labpath=examples%2Fscivision-core-functionality.ipynb
+   
+In brief, the notebook demonstrates the aforementioned five lines of code like so:
+
+1. Loading a pretrained (ImageNet) model, which was previously added to the scivision catalog with the name "scivision-test-plugin"
+    `model = load_pretrained_model('/path/to/model/repo')`
+2. Using the "default" scivision catalog to find a matching dataset, which the loaded model can be run on
+    `default_catalog.compatible_datasources(<model name>)`
+3. Loading the dataset in a format the model can recognise
+    `load_dataset('/path/to/data/repo')`
+4. Running the model on the data, performing simple model inference
+    `model.predict(<loaded data>)`
+
+If you counted just 4 lines of code there, well spotted! The fifth would be `default_catalog.compatible_models(<dataset name>)` which works in the same way as `compatible_datasources`, just reversed. See :ref:`api-docs` for more details.
```

Comment:
I found this is slightly confusing - five lines but then only four 'plus one'. Maybe the first is `import scivision`?

Reply:
I've made this less explicit by simply referring to "several lines of code"